tools/pd-ctl/pdctl/command/global.go

```diff
@@ -34,12 +35,25 @@ const (
 	pingPrefix        = "pd/api/v1/ping"
 )
 
+// PDCli is a pd HTTP client
+var PDCli pd.Client
```

Comment:
What about using a `sync.Once` here?

Reply:
because we support `-i` which can reset client addrs, maybe `sync.Once` is not suitable